package-lock.json

```diff
@@ -837,7 +837,8 @@
       "dependencies": {
         "ansi-regex": {
           "version": "5.0.0",
-          "resolved": "",
+          "resolved": "https://registry.npmjs.org/ansi-regex/-/ansi-regex-5.0.0.tgz",
```

Comment:
interesting, i think something may have went wrong with my previous npm audit PR then.

Reply:
I think this changed because i ran `npm i` locally which `when used without arguments, (npm i) it may write to package-lock.json to lock down the version of some dependencies if they are not already in this file`

https://stackoverflow.com/questions/52499617/what-is-the-difference-between-npm-install-and-npm-ci

Reply:
Interesting, I don't know how they weren't resolved in the first place. But looking at it closely, it looks like not versions actually changed, which would be my concern.